src/Evolution/Systems/GrMhd/ValenciaDivClean/ConservativeFromPrimitive.hpp

```diff
@@ -36,19 +36,21 @@ namespace ValenciaDivClean {
  * B_i \right) \\
  * {\tilde \tau} = & \sqrt{\gamma} \left[ \rho h W^2 - p - \rho W - \frac{1}{2}
  * (B^m v_m)^2 + \frac{1}{2} B^m B_m \left( 1 + v^m v_m \right) \right] \\
+ * {\tilde D_e} = & \sqrt{\gamma} \rho W Y_e \\
  * {\tilde B}^i = & \sqrt{\gamma} B^i \\
  * {\tilde \Phi} = & \sqrt{\gamma} \Phi
  * \f}
  *
  * where the conserved variables \f${\tilde D}\f$, \f${\tilde S}_i\f$,
- * \f${\tilde \tau}\f$, \f${\tilde B}^i\f$, and \f${\tilde \Phi}\f$ are a
- * generalized mass-energy density, momentum density, specific internal energy
- * density, magnetic field, and divergence cleaning field.  Furthermore
- * \f$\gamma\f$ is the determinant of the spatial metric, \f$\rho\f$ is the rest
- * mass density, \f$W = 1/\sqrt{1-v_i v^i}\f$ is the Lorentz factor, \f$h = 1 +
- * \epsilon + \frac{p}{\rho}\f$ is the specific enthalpy, \f$v^i\f$ is the
- * spatial velocity, \f$\epsilon\f$ is the specific internal energy, \f$p\f$ is
- * the pressure, \f$B^i\f$ is the spatial magnetic field measured by an Eulerian
+ * \f${\tilde \tau}\f$, \f${\tilde D_e}\f$, \f${\tilde B}^i\f$, and \f${\tilde
+ * \Phi}\f$ are a generalized mass-energy density, momentum density, specific
+ * internal energy density, electron fraction, magnetic field, and divergence
```

Comment:
@ffoucart is there a name for `tilde{D}_e` other than "electron fraction"? @kidder uses "densitized electron fraction" below. Does that sound reasonable?